qubes.SaltLinuxVM

```diff
@@ -29,7 +29,20 @@ read target_vm
 read salt_command
 
 rm -rf /srv
-mv /home/user/QubesIncoming/dom0/srv /srv
+user=$(qubesdb-read /default-user)
+if LC_ALL=C expr "x$user" : $'x[A-Za-z0-9_@.\x7F-\xFF][A-Za-z0-9_@.\x7F-\xFF-]*$' > /dev/null; then
```

Comment:
Please, keep it readable for mere mortals. If you are really afraid of `eval`-ing a dom0-provided value in a fresh DispVM, then don't use `eval`. Maybe `getent -- passwd "$user"` then?

Reply:
`getent` is way too complex.  Username validation is qubesd’s responsibility.  I’ll rip this out.